src/components/questions/index.cjsx

```diff
@@ -0,0 +1,26 @@
+React = require 'react'
+
+{TocStore, TocActions} = require '../../flux/toc'
+{CourseStore} = require '../../flux/course'
+LoadableItem = require '../loadable-item'
+
+
+Dashboard = require './dashboard'
```

Comment:
I think name this as `Dashboard` can lead to confusion, especially since we have another component we refer to as `Dashboard`...I think in general, each component should on be referred to with one name -- the one used to define it -- and be unique from each other.  Could this be `QuestionsDashboard` instead?